examples/02_run_openfast_cases/weis_driver_loads.py

```diff
@@ -3,22 +3,50 @@
 import sys
 
 from weis.glue_code.runWEIS     import run_weis
+from weis.glue_code.weis_args   import weis_args, get_max_procs, set_modopt_procs
 from openmdao.utils.mpi  import MPI
 
-## File management
-run_dir                 = os.path.dirname( os.path.realpath(__file__) )
-fname_wt_input          = run_dir + os.sep + 'IEA-15-240-RWT.yaml'
-fname_modeling_options  = run_dir + os.sep + 'modeling_options_loads.yaml'
-fname_analysis_options  = run_dir + os.sep + 'analysis_options_loads.yaml'
+# Parse args
+args = weis_args()
 
+## File management
+run_dir = os.path.dirname( os.path.realpath(__file__) )
+fname_wt_input = os.path.join(run_dir, 'IEA-15-240-RWT.yaml')
+fname_modeling_options = os.path.join(run_dir, 'modeling_options_loads.yaml')
+fname_analysis_options = os.path.join(run_dir, 'analysis_options_loads.yaml')
 
 tt = time.time()
-wt_opt, modeling_options, opt_options = run_weis(fname_wt_input, fname_modeling_options, fname_analysis_options)
+maxnP = get_max_procs(args)
+
```

Comment:
Do we want the double-run when MPI is used to be the default for all examples?  Or should users do a dry-run and update the modeling options themselves?  We're demonstrating both here, which is also fine.

Reply:
I found both use cases useful. expert users might want to know the size of their problem, whereas newer users may just want to run